Automattic/wp-calypso
# Jetpack Scan: Remove Polling Request Limit

Resolves https://github.com/Automattic/wp-calypso/issues/96245

## Proposed Changes

* Removes the `MAX_RETRY_COUNT` configuration for the Scan API requests.
* Adjusts the scan status rendering conditional to preserve fixes made surrounding to the order of rendering and preventing flickers between scan states.

## Why are these changes being made?


* See https://github.com/Automattic/wp-calypso/issues/96245

## Testing Instructions



* Start with a site that has access to Jetpack Scan.
* Install a vulnerable plugin such as WooCommerce 3.0.0.
* Navigate to Jetpack > Scan and start a new scan to detect the threat.
* Start an auto-fixer for the threat.
* Open the Network tab and refresh the page.
* Observe the `scan` requests, ensuring it re-requests as long as a threat has a `fixer_status` of `in_progress`.
* Validate that the placeholder/loading/results views are shown in proper order without any UI flicker throughout the entire process.

## Pre-merge Checklist
